packages/twenty-front/src/modules/activities/tasks/components/TaskRow.tsx

```diff
@@ -56,7 +56,7 @@ const StyledDueDate = styled.div<{
 }>`
   align-items: center;
   color: ${({ theme, isPast }) =>
-    isPast ? theme.font.color.danger : theme.font.color.secondary};
+    isPast ? theme.color.red : theme.font.color.secondary};
```

Comment:
this should not be necessary if we keep font.color.danger = color.red, see comment below